### ISSUE TITLE
Validation in form fields

### ISSUE THREAD

**Issue by chrisphillers** [adobe/react-spectrum#1312]
# 🙋 Feature Request

Given you have validation props in your form and other components it would be great if React Spectrum handled validation props (eg error messages) when the value in the field/component isn't valid.

## 🤔 Expected Behavior

Handle validation messages/errors - layout/style in an Adobe Spectrum design.

## 😯 Current Behavior

There isnt this feature - currently need to create our own.

## 💁 Possible Solution

Would be amazing if you could pass in a validationMessage prop

## 🔦 Context

I am creating a form using React Spectrum am managing validation and would love validation messages to work and look like the rest of the form.

**Comment by snowystinger** [comment]
Just to clarify, you mean messages like this? https://spectrum.adobe.com/page/text-field/#Help-text-(description-and-error-message)
If so, this was a recent addition from our design system, I'll try to get some more information on timeline for when the team would expect to work on it or if it's finished enough that we could accept a contribution.

**Comment by snowystinger** [comment]
Update, we're waiting on our design system tokens before we can implement this. I do not know the timeline for that, but am trying to find out.

**Comment by devongovett** [comment]
Closing in favor of #1346.
